src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -752,6 +763,26 @@ public static unsafe Vector128<byte> Create(byte e0, byte e1, byte e2, byte e3,
                 return Sse2.UnpackLow(lo64, hi64).AsByte();                                         // <  0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15 >
             }
 
+            if (AdvSimd.IsSupported)
+            {
+                Vector128<byte> result = CreateScalarUnsafe(e0);
```

Comment:
@kunalspathak You might want to add comment here in the same fashion as it's done for Sse2 case

Reply:
Spoke offline and this is not needed.